clairmeta/dcp_check_global.py

```diff
@@ -7,7 +7,7 @@
 from clairmeta.dcp_utils import list_cpl_assets, cpl_probe_asset
 from clairmeta.dcp_check import CheckerBase
 from clairmeta.utils.sys import all_keys_in_dict
-
+from clairmeta.profile import DCP_CHECK_PROFILE
```

Comment:
You can remove this now.

Reply:
Oh yes. I forgot to tidy up the room after playing. That should do.